runtime/bindings-generator.cs

```diff
@@ -1915,6 +1915,18 @@ static IEnumerable<FunctionData> GetFunctionData ()
 				}
 			);
 
+			data.Add (
+				new FunctionData {
+					Comment = " // Matrix4 func (/* UIInterfaceOrientation */ nint)",
+					Prefix = "simd__",
+					Variants = Variants.All,
+					ReturnType = Types.Matrix4f,
+					Parameters = new ParameterData[] {
+						new ParameterData { TypeData = Types.NInt },
```

Comment:
@rolfbjarne why is `NInt` works here ?
since it did not for the previous declaration, which only builds fine if Int32 and Int64 are used ?